packages/hooks/src/useWithdrawLocked.ts

```diff
@@ -0,0 +1,17 @@
+import { useStore } from '@deriv/stores';
+import useNeedPOI from './useNeedPOI';
+
+const useWithdrawLocked = () => {
+    const { client } = useStore();
+    const is_need_poi = useNeedPOI();
+
+    if (!client.account_status?.status) {
+        return false;
+    }
+
+    const status_name_locked = client.account_status?.status?.some(status_name => status_name === 'withdrawal_locked');
+
+    return status_name_locked || is_need_poi || false;
```

Comment:
```suggestion
    const has_withdrawal_locked_status = client.account_status.status.some(status => status === 'withdrawal_locked');
    const is_withdrawal_locked = has_withdrawal_locked_status || is_need_poi;

    return is_withdrawal_locked;
```

Reply:
@farzin-deriv It's a good suggestion to remove the extra `false` value, but I think adding another variable for storing the return value will add no benefits to the code, also it may decrease the readability of the code.